orchestrator/gravity_utils/src/connection_prep.rs

```diff
@@ -26,6 +26,7 @@ pub struct Connections {
 /// operation in a error resilient manner. TODO find some way to generalize
 /// this so that it's less ugly
 pub async fn create_rpc_connections(
+    chain_prefix: String,
```

Comment:
Can we name this address_prefix and do so throughout the PR. cosmos_prefix and chain_prfix are a bit confusing

Reply:
the correct term would be `bech32_hrp` (human readable prefix)

Reply:
rename done. We'll be able to remove it as an argument pretty soon. There's upstream work to allow prefixes to be simply queried by the library.